k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-ha/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: CORTX data
```

Comment:
Don't know but I think 
this line can be CORTX ha
and the name in below line (3) can be cortx-ha

Reply:
Yes, thanks for catching this. Fixed.